plugins/techdocs-backend/src/service/router.ts

```diff
@@ -106,11 +106,16 @@ export async function createRouter({
   });
 
   router.get('/docs/:namespace/:kind/:name/*', async (req, res) => {
-    const { kind, namespace, name } = req.params;
     const storageUrl =
       config.getOptionalString('techdocs.storageUrl') ??
       `${await discovery.getExternalBaseUrl('techdocs')}/static/docs`;
 
+    res.redirect(`${storageUrl}${req.path.replace('/docs', '')}`);
+  });
```

Comment:
This redirect of requests from `/docs/*` to `/static/docs/*` is no longer useful. TechDocs Frontend can directly request to `techdocs.storageUrl` or `/static/docs` on TechDocs backend.

Reply:
@OrkoHunter how does one retrieve techdocs.storageUrl from frontend? a little unsure

Reply:
Updated as described